vaas/vaas/settings/base.py

```diff
@@ -277,3 +277,8 @@ def generate_redis_url(hostname: str, port: int, db_number: int, password: Optio
 ROUTES_CANARY_HEADER = env.str('ROUTES_CANARY_HEADER', default='x-canary-random')
 if ROUTES_CANARY_HEADER:
     ROUTES_LEFT_CONDITIONS[f'std.real(req.http.{ROUTES_CANARY_HEADER},0)'] = 'Canary'
+
+DOMAIN_MAPPER = env.json('DOMAIN_MAPPER', default={
+    'example': 'example.com',
+    'example.pl': 'example.{{ PLACEHOLDER }}.pl',
```

Comment:
AFAIR we tried that approach in ROUTES_LEFT_CONDITIONS and it didn't work properly because keys containing dots were interpreted as nested keys.

Reply:
Yes. I forgot that we will have dots on keys. I change it like ROUTES_LEFT_CONDITIONS.